Firefall DISASM Name Manager/Firefall DISASM Name Manager/MainForm.cs

```diff
@@ -155,15 +168,21 @@ public int AddNamesToListView(List<NameClass> NamesObject, bool UpdateFullDuplic
                 DeduplicationForm deduplicationForm = new DeduplicationForm(DuplicateObjects, SourceObjects);
                 if (deduplicationForm.ShowDialog(this) == DialogResult.OK)
                 {
-                    // TODO: Currently this is adding another copy of the "deduplicated" name to the list of what to add
-                    // This instead needs to be updating the exist entry
+                    // TODO: Currently this is adding another copy of the "deduplicated" name to the list of what to
+                    // add. This instead needs to be updating the exist entry.
```

Comment:
Move "add." back up to the end of the previous line.
Change "updating the exist entry" to say "updating the existing entry" to fix the typo originally made.

Reply:
Done.